05-onchain.md

```diff
@@ -513,7 +513,14 @@ A local node:
   using the revocation private key.
   - SHOULD extract the payment preimage from the transaction input witness, if
   it's not already known.
-  - MAY use a single transaction to *resolve* all the outputs.
+  - if `option_anchor_outputs` applies:
+  - otherwise:
+    - MAY use a single transaction to *resolve* all the outputs.
+    - if confirmation doesn't happen before reaching `security_delay` blocks from
+  expiry:
+      - MUST *resolve* the _local node's main output_ in its own penalty transaction
+  as if a single resolving-transaction has been previously used its propagation might be
+  under pinning.
```

Comment:
nit:

```suggestion
      - MUST *resolve* the _local node's main output_ in its own, separate penalty transaction.
      A previous penalty transaction claiming multiple revoked outputs at once may be blocked
      from confirming because of a transaction pinning attack.
```